### PR TITLE
[Docs] Add name key to config.json

### PR DESCRIPTION
**Pull request by SaschaMann**

### CODE REVIEW

**Review by cmcaine — APPROVED** [review]

File: docs/maintainers/migrating-your-config-json-files.md [hunk 1]
```diff
@@ -48,3 +48,11 @@ jq 'del(.foregone)
...
 
 [jq]: https://stedolan.github.io/jq/
+
+## New keys
+
+<!-- TODO: Properly document the new config.json format -->
+
+Exercises may contain the following new keys:
+
+- `name`: The title of the exercise if it differs from the titleised slug (optional)
```
Comment (SaschaMann) [inline docs/maintainers/migrating-your-config-json-files.md:58]:
```suggestion
- `name`: The title of the exercise if it differs from the titleised slug
```

**Comment by SaschaMann** [comment]
(Merged before it gets out of sync again. If you want the doc change to be changed, let me know I'll open a follow-up PR)

**Pull request merged**